src/components/Booking.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import { format } from 'date-fns';
+import { Select, Option } from '@material-tailwind/react';
+import { bookCar } from '../redux/Reservations/reservationsSlice';
+import api from '../api/api';
+import { useAuthUser } from '../redux/Auth/useAuthUser';
+import { allCars } from '../redux/Home/home';
+
+const Booking = () => {
+  const [pickupDate, setPickupDate] = useState(null);
+  const [returnDate, setReturnDate] = useState(null);
+  const user = {
+    email: 'abc@gmail.com',
+    password: '123456',
+  };
+  api.login(user);
```

Comment:
Please remove this line because we logged in by using logged form make sure this line of code will remove from there :face_exhaling:

Reply:
Oh yeah, I forgot to remove to that 😅